web/modules/custom/usagov_ssg_postprocessing/usagov_ssg_postprocessing.module

```diff
@@ -70,29 +74,29 @@ function usagov_ssg_postprocessing_page_attachments_alter(array &$attachments) {
 /**
 * Static Site Generation Toggle State Form ID
 */
-function usagov_ssg_postprocessing_get_static_state_form_id() {
+function usagov_ssg_postprocessing_get_static_state_form_id(): string {
   return 'toggle_static_site_generation_form';
 }
 
 /**
 * Static Site Generation Toggle State Form Button Name
 */
-function usagov_ssg_postprocessing_get_static_state_button_name() {
+function usagov_ssg_postprocessing_get_static_state_button_name(): string {
   return 'confirm_toggle';
 }
 
 /**
  * Toggle State variable name for sset/sget/sdel
  */
-function usagov_ssg_postprocessing_get_static_state_var() {
+function usagov_ssg_postprocessing_get_static_state_var(): string {
   return 'usagov.tome_run_disabled';
 }
 
 /**
   * Implements hook_form_alter().
   * Static Site Generation form text updates.
   */
-function usagov_ssg_postprocessing_form_alter(&$form, &$form_state, $form_id) {
+function usagov_ssg_postprocessing_form_alter(mixed &$form, mixed &$form_state, string $form_id): void {
```

Comment:
We can avoid `mixed` here, see <https://github.com/usagov/usagov-2021/wiki/PHPStan-Tips#form-alter-hooks>